kak-lsp.toml

```diff
@@ -149,7 +149,12 @@ roots = ["package.json"]
 command = "json-languageserver"
 args = ["--stdio"]
 
-# Requires Julia packages "LanguageServer", "StaticLint" and "SymbolServer"
+# Requires Julia packages "LanguageServer"
+# Run: `julia --project=@kak-lspconfig -e 'import Pkg; Pkg.add("LanguageServer")'` to install it
```

Comment:
so this means we give precedence to the LanguageServer package installed to ~/.julia/environments/kak-lspconfig?
Why is that better than installing LanguageServer without the --project flag?

A non-obvious manual step is not great.
Also, ~/.julia/environments/kak-lsp would be a more fitting name.

Reply:
I think this is the best way to avoid deps from other packages conflicting each other. There are instances where LanguageServer.jl and another package depends on the same dependency and the other one requires a version lower and may break some functionalities with LanguageServer.jl. It is better for it to be in a separate environment.

Reply:
Also, if the user does not have `LanguageServer.jl` in `~/.julia/environments/kak-lsp`, it will look for `LanguageServer.jl` in the default global environment.